airflow/jobs/scheduler_job.py

```diff
@@ -686,10 +664,10 @@ def _process_dags(self, dagbag, dags, tis_out):
         :type dagbag: airflow.models.DagBag
         :param dags: the DAGs from the DagBag to process
         :type dags: airflow.models.DAG
-        :param tis_out: A list to add generated TaskInstance objects
-        :type tis_out: list[TaskInstance]
-        :rtype: None
+        :return: A list of TaskInstance objects
+        :rtype: list[TaskInstance]
```

Comment:
Can you also add rtype for _process_task_instances also? Now it is difficult to check if this is true. Especially since this method previously use TaskInstanceKeyType.